README.md

```diff
@@ -367,9 +367,8 @@ Optional:
 
 ### Services
 
-You will need Docker Desktop with the Docker MCP extension installed for
```

Comment:
I'm puzzled as to why changes like this are showing up in this PR as they are not conflicts